draft-ietf-quic-multipath.md

```diff
@@ -1113,25 +1045,19 @@ multipath extension is used and an endpoint has multiple addresses that
 could lead to switching between different paths, it should rather maintain
 multiple open paths instead.
 
-If an endpoint uses a new connection ID after an idle period
-and a NAT rebinding leads to a 4-tuple changes on the same packet,
-the receiving endpoint may not be able to associate the packet to
-an existing path and will therefore consider this as a new path.
-This leads to an inconsistent view of open paths at both peers,
-however, as the "old" path will not work anymore, it will be silently
-closed after the idle timeout expires.
```

Comment:
This was removed entirely as this problem does not exist anymore with an explicit path ID.